sql/branch.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
+    -- Tips that have been set to zero should have zero dues.
+
+    UPDATE payment_instructions SET due = 0 WHERE amount = 0 AND due != 0;
```

Comment:
P.S. Do we do this normally? When I drop my tip to zero, does my due get reset as well?